docs/gitbook/examples/to-do-list.md

```diff
@@ -1,425 +1,257 @@
 
-This To Do List example is setup as an interactive tutorial. If you want to skip ahead to the interactive part of the tutorial, [go ahead](to-do-list.md#adding-toggle-functionality).
+Directory | Purpose
+----------|-----------
+`public`                  | Website's public directory
+`src`                     | Source code for the app
+`src/adapters`            | A set of adapters to fill UI details for the covenant forms
+`src/components`          | React components
+`src/constants`           | A file of constants that are used throughout the application (Ex. URLs)
+`src/containers`          | React components that are connected to redux state
+`src/css`                 | The css
+`src/interbit`            | All of the covenants needed for this application
+`src/redux`               | Reudcers and any other redux related files
+`tests`                   | Jest tests
+`App.js`                  | The main React component that loads the SPA
+`exports.js`              | A file to export anything that may be needed outside of this project (Ex. covenant action creators)
+`index.js`                | The file that creates the Redux store, attaches the [Interbit middleware](../reference/interbit-middleware/README.md), and runs the SPA
+`registerServiceWorker.js` | A file used by create-react-app to register a [service worker](https://developers.google.com/web/fundamentals/primers/service-workers/)
+`interbit.config.js`      | The [Interbit configuration file](../reference/interbit-cli/config.md)
+`interbit.prod.config.js` | The production environment interbit configuration file
+`keyPair.js`              | A file to import a public private key pair from environment variables when in production
+`static.json`             | Used to serve a single page application (SPA) in Heroku. This describes where to fallback if routes are not matched (Returning 404)
 
-In this example,
-1. we will step through the components of the example as it is now, and address what each component does, and how each works.
-  1. [Actions](to-do-list.md#actions)
-  1. [Smart Contract](to-do-list.md#smart-contract)
-  1. [Testing](to-do-list.md#testing)
-  1. [Script](to-do-list.md#script)
-1. we will add new code to each of the sections covered in order to add toggling functionality to the To Do List.
-  1. [Adding an Action](to-do-list.md#adding-a-new-action)
-  1. [Handling the Action](to-do-list.md#handling-the-action-in-the-smart-contract)
-  1. [Testing the Action](to-do-list.md#testing-the-action)
-  1. [Dispatching the Action](to-do-list.md#dispatching-the-action-to-the-chain)
 
+## Walk Through
```

Comment:
This compound word is better written as "Walk-through". Otherwise, interpreting the title as a sentence means that the noun is missing; walk through _what_?